cmdeploy/src/cmdeploy/service/chatmail-metadata.service.f

```diff
@@ -5,6 +5,41 @@
 ExecStart={execpath} /run/dovecot/metadata.socket vmail /home/vmail/mail/{mail_domain}
 Restart=always
 RestartSec=30
+User=vmail
```

Comment:
if we do this, do we still need to pass in `vmail` to the metadata script and do the "chown" shenenigans in https://github.com/deltachat/chatmail/blob/main/chatmaild/src/chatmaild/metadata.py ?